bin/ColorScript/Sources/ColorScriptCore/Sources/ColorScriptCore.swift

```diff
@@ -0,0 +1,118 @@
+// swiftlint:disable force_unwrapping
+import Foundation
+
+extension Dictionary {
+  public func withAllValuesFrom(_ other: Dictionary) -> Dictionary {
```

Comment:
Swift now has a built-in function for this, called `merging(_:uniquingKeysWith:)` you can reproduce the same behaviour as the function in this extension by doing:
```swift
public func withAllValuesFrom(_ otherDictionary: Dictionary) -> Dictionary {
  return self.merging(otherDictionary) { $1 }
}
```
If it's used in more than one place, I'd do it this way, otherwise just use that directly at the callsite.

Reply:
Good to know!